src/screens/InitWallet.js

```diff
@@ -245,8 +245,9 @@ class LoadWordsScreen extends React.Component {
   }));
 
   onChangeText = (text) => {
-    const words = text.trim().split(' ');
+    const words = text.trim().split(/[ \n\t]/);
```

Comment:
Why not go all the way then, and accept any whitespace with `\s`.

Reply:
Good one. Fixed. Thanks!